markovs_household/data/household.py

```diff
@@ -0,0 +1,57 @@
+from enum import Enum
+from dataclasses import dataclass
+from typing import Dict
+
+from markovs_household.data.appliance import Appliance
+
+
+class HouseHoldType(Enum):
+    """
+    Enumeration of different household types
+    """
+    FLAT = "flat"
+    HOUSE = "house"
+
+
+@dataclass(frozen=True)
+class Household:
+    """
+    Represents a household that houses a certain amount of appliances
+    :var appliances: mapping of an appliance to the number of the particular appliance within the household

```

Comment:
Python documentation seems not to be correct. Is a `var` right here?

Reply:
This does not really work with dataclasses. The idea is that given how they are structured they should be more or less self explanatory so I have removed the arguments descriptions.